test/ViewsTest.cpp

```diff
@@ -265,3 +265,91 @@ TEST(Views, verifyLineByLineWorksWithChunksBiggerThanLines) {
   ++iterator;
   ASSERT_EQ(iterator, lineByLineGenerator.end());
 }
+
+// _____________________________________________________________________________
+TEST(Views, RvalueView) {
+  // A simple struct that knows if it has been moved from.
+  struct MoveTracker {
+    bool wasMoved_ = false;
+    MoveTracker() = default;
+    MoveTracker(const MoveTracker&) = default;
+    MoveTracker& operator=(const MoveTracker&) = default;
+    MoveTracker(MoveTracker&& rhs)
+        : wasMoved_{std::exchange(rhs.wasMoved_, true)} {}
+    MoveTracker operator=(MoveTracker&& rhs) {
+      wasMoved_ = std::exchange(rhs.wasMoved_, true);
+      return *this;
+    }
+  };
+
+  std::vector<MoveTracker> vec;
+  vec.resize(13);
+
+  std::vector<MoveTracker> target;
+  {
+    // Move the first 5 elements via an `RvalueView` + ql::ranges::copy.
+    auto view = ad_utility::RvalueView{vec};
+    static_assert(ql::ranges::random_access_range<decltype(view)>);
+    ASSERT_EQ(view.size(), 13);
+    ql::ranges::copy(view | ql::views::take(5), std::back_inserter(target));
+    ASSERT_EQ(target.size(), 5);
+    for (size_t i = 0; i < vec.size(); ++i) {
+      ASSERT_EQ(i < 5, vec[i].wasMoved_);
+    }
```

Comment:
If you create an `ElementsAre(true, true, true, true, true, false, false, false, false, false, false, false, false)` matcher, mapped using `AD_MEMBER`,  any failures should become easier to read (alternatively using `ElementsAreArray` which allows you to avoid typing everything out manually) and makes the code shorter in 3 occurrences.